### ISSUE TITLE
Could canvasXpress respect axis lines and ticks in ggplot2 better?

### ISSUE THREAD

**Issue by davidhodge931** [neuhausi/canvasXpress#147]
```{r} 
library(tidyverse)
library(palmerpenguins)
library(canvasXpress)

p <- penguins |> 
	ggplot() +
	geom_point(aes(x = flipper_length_mm, 
								 y = body_mass_g, 
								 colour = species,)) +
	theme_classic()

p

canvasXpress(p)
```

![image](https://github.com/neuhausi/canvasXpress/assets/55065152/77597f28-7103-495d-9ae5-4c6b48e1c497)

**Comment by neuhausi** [comment]
Should be an easy fix. I will include it in the next version

**Comment by neuhausi** [comment]
Fixed in 48.6. I will leave the issue open for you to close it after the new version is released.

![cX-htmlwidget-aab856462fe0e136a77e-cx](https://github.com/neuhausi/canvasXpress/assets/254923/f75cb1ee-7be7-4a28-842a-32133279bb18)

**Comment by davidhodge931** [comment]
Works great for `axis.line` and `axis.ticks` theme elements.

Could this also work for the lower level theme elements?
      axis.line.y.left 
        axis.line.y.right
        axis.ticks.y.left 
        axis.ticks.y.right
          axis.line.x.top 
        axis.line.x.bottom
        axis.ticks.x.top 
        axis.ticks.x.bottom

**Comment by neuhausi** [comment]
I thought I did. Can you please provide example?

**Comment by neuhausi** [comment]
There was in fact a bug that it will be fix in 48.7 I added this example.

```
library(ggblanket)
library(ggplot2)
library(dplyr)
library(stringr)
library(tidyr)
library(palmerpenguins)
library(patchwork)

p = penguins |>
  gg_point(
    x = flipper_length_mm,
    y = body_mass_g,
  )

canvasXpress(p)
```
![cX-htmlwidget-27a38f5570aac8739c6b-cx](https://github.com/neuhausi/canvasXpress/assets/254923/45b983e2-9610-41ff-aa8c-807b68bafbdf)

**Comment by davidhodge931** [comment]
Nice :)
